composer.json

```diff
@@ -25,7 +26,7 @@
         "doctrine/coding-standard": "^6.0",
         "doctrine/dbal": "^2.5.4",
         "doctrine/mongodb-odm": "^1.3.0",
-        "doctrine/orm": "^2.5.4",
+        "doctrine/orm": "^2.7.0",
```

Comment:
Why are we bumping this?

Reply:
It's in order to make sure we get the changes from this PR: https://github.com/doctrine/orm/pull/7079 (`getJoinTableName` is called inside `getAssociationTables`)

Reply:
Why is that relevant in this PR?

Reply:
Because the tests will not pass otherwise. Do you want me to extract this commit in a separate PR?

Reply:
Nah it's allright it can be here, it's small